### PR TITLE
[AC-9353] Startup Profile - new fields and layout changes

### PR DESCRIPTION
**Pull request by shakirandagire**
Changes made in [AC-9353](https://masschallenge.atlassian.net/browse/AC-9353):
Add new fields to the model.

How to test:
- New fields are added to the Startup model.
- Migrations for the changes exist.

### CODE REVIEW

File: accelerator/migrations/0087_update_startup_profile.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        ),
+        migrations.AlterField(
+            model_name='coreprofile',
+            name='preferred_name',
+            field=models.CharField(
+                blank=True,
+                max_length=32,
+                null=True,
+                verbose_name='Nickname / Preferred Name'),
+        ),
```
Comment (iankigen) [inline accelerator/migrations/0087_update_startup_profile.py:58]: is there a reason we have core profile model changes

**Review by iankigen — APPROVED** [review]
Code changes look good
- A startup / enterprise flag: boolean field is_startup ✅ 
- Street address of my company’s physical location ✅ 
- ethnographic and historical information on my founders, boolean flags for each of:
  - BIPOC Founder: bipoc_founder ✅ 
  - female or transgender Founder: female_or_transgender_founder ✅ 
  - first-time Founder: first_time_found ✅

File: accelerator/migrations/0087_update_startup_profile.py (hunk 1 above)
Comment (shakirandagire) [reply accelerator/migrations/0087_update_startup_profile.py:58]: Let me look at the migration. I am unsure why this is the case

Comment (AmosWels) [reply accelerator/migrations/0087_update_startup_profile.py:58]: the coreprofile field alteration on `preferred_name` exists but I see it makes no change to the field

**Review by AmosWels — APPROVED** [review]
changes look good and migrations run successfully

**Pull request merged**